src/components/catagories.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import '../App.css';
+
+const Categories = () => (
+  <>
+    <div>
+      <button type="button">Check Status </button>
+    </div>
+  </>
```

Comment:
- [ ] According to requirements, this `<Categories>` component should contain a `<Navbar>` component, which has 2 `<Link>`s. One to `/` and one to `/categories`. Please implement this change 🙏

Reply:
Hello @codepantha, according to instructions category component contains only the check status button. Please correct me if I am wrong.